### DISCUSSION TITLE
I quit my job and all I have is a massСode.

### DISCUSSION THREAD

**Discussion by antonreshetov** [massCodeIO/massCode#145] [Announcements]
Hi there. 

Three months ago I quit my job, and devoted this time to creating v2 from scratch. I am constantly thinking about the app and how to improve it. I get a lot of positive feedback that the app helps them speed up development a bit and I'm glad I was able to make a useful tool for people.  

If you find the app useful, please [donate or sponsor](https://masscode.io/donate) it. If 1/3 of the massCode users donate just $1 a month, that would be a nice monthly salary. 

I don't want to go into SaaS such as cloud storage of snippets or any other paid solutions. I want this project to remain completely open source. There are fewer and fewer projects left today that wouldn't go into the cloud. And I understand them very well, food is not free. 

Thank you to everyone who has already donated. This is very valuable to me and inspires me to keep this project going.

**Comment by sangyuxiaowu** [comment]
I saw this project on the Chinese social platform. It's a very good project, which I wanted to do before. Look forward to internationalization.

**Reply by pkd2989** [reply]
massCode has great potential to be the most useful/productive Snippet manager. 
I will pay for a license, if I can get a specific feature (I also created a discussion on this). 
I should get my snippet quickly irrespective of the OS / IDE/ Editor, I am using. For this, we need to have a quick launcher for search and copy.
Also if possible link it to Note-taking apps like Joplin, Notable etc. 

Thanks,

**Reply by antonreshetov** [reply]
> I will pay for a license, if I can get a specific feature...

I don't sell licenses, it's all voluntary donation. I also don't do custom development of "X features for X needs for X people". I have a certain perspective on the app. I certainly listen to interesting ideas, thanks for them. But I also think that even in its current form the app is worth at least a dollar in recognition of the effort.

**Reply by antonreshetov** [reply]
> I saw this project on the Chinese social platform. It's a very good project, which I wanted to do before. Look forward to internationalization.

In the next version I will try to add localization. I will need help with translation, of course.

**Reply by sangyuxiaowu** [reply]
> 
Of course, everyone will be very happy to help masscode do better. After internationalization, you will get more recommendations on the local social platforms of various countries, and achieve the goal of stars faster.
Or you can provide payment methods in Chinese. We usually use Alipay and WeChat to scan and pay.

**Comment by pkd2989** [comment]
There is not even a question for recognition of the effort, its 1$ or whatever.
What I mentioned is not for asking a custom development, its one of the feature what I felt has been missing from most of the snippet managers I have tried, but which can be very useful. 
Most snippet manager do a good job in storing and organizing the snippets, but can assist with developer's workflow in a better way. The reason for massCode having IDE specific packages/plugins for vscode etc., is stemming from this need itself. 

Good luck with your journey.

**Comment by wapoa** [comment]
Added monthly contribute. Appreciate the great product and the vision of keeping it free. I hope more people contribute if they use this.

**Reply by antonreshetov** [reply]
> Added monthly contribute. Appreciate the great product and the vision of keeping it free. I hope more people contribute if they use this.

Thank you a lot 🙏
